config/alphagov.yml

```diff
@@ -322,6 +322,17 @@ govuk-publishing-experience:
   compact: true
   <<: *common_properties
 
+
+govuk-publishing-on-platform-content:
+  channel: '#govuk-publishing-on-platform-tech'
+  compact: true
+  <<: *common_properties
+  repos:
+    - publisher
+    - collections-publisher
+    - content-tagger
+    - smart-answers
```

Comment:
```suggestion
```

These aren't needed if the team is set up as an owner in the developer docs. So if you modify https://github.com/alphagov/govuk-developer-docs/pull/4018/files to use the `#govuk-publishing-on-platform-tech` channel then these will be determined automatically.

Reply:
Setting them explicitly as I wanted to have Smart Answers too but we aren't taking formal ownership _yet_ - is it an append or a replace (>> or >)?

Reply:
Sure, that'd make sense. It works as a replace, so if you set any here it'll override the automatic loading: https://github.com/alphagov/seal/blob/42e7f535984e48b5d87ca2affc0c036716329b2d/lib/team_builder.rb#L48-L50

It would be a little weird you getting them when you don't own it as the same ones will be going to the owning team and the seal will be encouraging you to review PRs that you probably shouldn't be reviewing until you own it.